#! /bin/sh /usr/share/dpatch/dpatch-run
## 16_hal_shutdown.dpatch by Michael Haas <laga+mythtv@laga.ath.cx>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change some defaults to work better with Ubuntu:
## DP: For reboot and shutdown, we can just talk to HAL


--- a/programs/mythfrontend/globalsettings.cpp	(revision 16838)
+++ b/programs/mythfrontend/globalsettings.cpp	(working copy)
@@ -2194,7 +2194,7 @@
 {
     HostLineEdit *ge = new HostLineEdit("RebootCommand");
     ge->setLabel(QObject::tr("Reboot command"));
-    ge->setValue("reboot");
+    ge->setValue("/usr/share/mythtv/myth-reboot.sh");
     ge->setHelpText(QObject::tr("Command or script to run if you select "
                     "the reboot option from the exit menu, if the option "
                     "is displayed. You must configure an exit key to "
@@ -2206,7 +2206,7 @@
 {
     HostLineEdit *ge = new HostLineEdit("HaltCommand");
     ge->setLabel(QObject::tr("Halt command"));
-    ge->setValue("halt");
+    ge->setValue("/usr/share/mythtv/myth-halt.sh");
     ge->setHelpText(QObject::tr("Command or script to run if you select "
                     "the shutdown option from the exit menu, if the option "
                     "is displayed. You must configure an exit key to "
